src/sentry/static/sentry/app/views/organizationIntegrations/integrationViewController.tsx

```diff
@@ -1,13 +1,26 @@
 import React from 'react';
-
 import {isIntegrationDirectoryActive} from 'app/utils/integrationUtil.tsx';
+import withOrganization from 'app/utils/withOrganization';
+import {RouteComponentProps} from 'react-router/lib/Router';
+import {Organization} from 'app/types';
 
 import Control from './index';
 import Test from './integrationListDirectory';
 
-export default function IntegrationViewController(props) {
-  if (isIntegrationDirectoryActive()) {
+type Props = RouteComponentProps<{orgId: string}, {}> & {
+  organization: Organization;
+  hideHeader: boolean;
+};
+
+function IntegrationViewController(props: Props) {
+  if (
+    isIntegrationDirectoryActive(
```

Comment:
@NisanthanNanthakumar I think it would be easier if `isIntegrationDirectoryActive` just took an argument for `organization` (which could be undefined) so we don't have to repeat `organization?.experiments?.IntegrationDirectoryExperiment` in multiple places.